packages/node-sdk/src/client.ts

```diff
@@ -611,9 +504,140 @@ export class BucketClientClass {
         key: key,
         targetingVersion: features[key].targetingVersion,
         evalResult: features[key].isEnabled,
+      }).catch((err) => {
+        this._config.logger?.error(
+          `failed to send check event for "${key}": ${err}`,
+          err,
+        );
       });
 
-      return features[key].isEnabled;
+      const feature = features[key];
+
+      return {
```

Comment:
is the `maskedProxy` smart to ignore the `track()` property?

Reply:
no. It'll send the "check" event even if you call `track`. I think it's OK for now because it stil indicates that there's a reference in the code.